extensions/amp-sidebar/0.1/amp-sidebar.css

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+amp-sidebar {
+  position: fixed !important;
+  top: 0 !important;
+  max-height: 100vh !important;
+  height: 100vh;
+  max-width: 80vw !important;
+  background-color: #efefef;
+  min-width: 30vw !important;
+  overflow-x: hidden !important;
+  overflow-y: auto !important;
+  z-index: 9999 !important;
+  will-change: transform;
+  -webkit-overflow-scrolling: touch;
+}
+
+amp-sidebar[direction="left"] {
+  left: 0 !important;
```

Comment:
:) since this is a sidebar  - it is understandable that it would be on the sides. again this could be achieved by margin tricks as we are not being strict on that